### DISCUSSION TITLE
Previous/next note in folder

### DISCUSSION THREAD

**Discussion by rotane** [chrisgurney/obsidian-note-toolbar#47] [Ideas]
I like the idea of the previous/next daily note command. However, since i don't use daily notes but have a rather large archive of journal entries, i'd like to be able to quickly navigate back and forward. Is there a way to add a command that opens previous/next file in folder?

**Comment by chrisgurney** [comment]
I'm sure there must be a plugin that provides this. 

Here's a thread that might have some pointers:
https://www.reddit.com/r/ObsidianMD/comments/11i0n7r/navigate_to_previousnext_note_in_a_directory/

**Reply by rotane** [reply]
Thanks for the heads-up! Didn't find it there, but found another plugin in the process of diggin around that adds these 2 commands that i can now use: https://github.com/FabianUntermoser/obsidian-neighbouring-files-plugin

Cheers!
